RELEASE_CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
-## [v3.0.0] - 2024-03-04
```

Comment:
Why did you remove this update date?

Reply:
Some release version (below v0.7) doesn't have date. So I unified the subject style for parsing. I thought we can find the date at release note or commit. If you need to maintain the date format, we need to change previous release title.

Reply:
We did not release as public below v0.7. And we did not count it before. But we counted the release date after public. So how about remaining the histories?